tests/test_dojo.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from App.dojo import Dojo
+
+class TestDojo(unittest.TestCase):
+    '''
+    Test Fixture: a piece of code that can construct and configure
+    the system and the tasks getting ready to be tested and then cleans up
+    '''
+    def setup(self):
+        self.dojo = Dojo()
+
+
+    def test_create_room_successfully(self):
+        
+        self.assertEqual(len(self.dojo.all_rooms), 0)
+        self.dojo.create_room('Blue', 'office')
+        self.assertEqual(len(self.dojo.all_rooms), 1, 'rooms added successfully')
+
+    def test_create_multiple_offices(self):
+        self.assertEqual(len(self.dojo.all_rooms), 0)
+        self.dojo.create_room('Blue', 'office')
+        self.dojo.create_room('Black', 'office')
+        self.dojo.create_room('Brown', 'office')
+        self.assertEqual(len(self.dojo.all_rooms), 3, 'rooms added successfully')
+        
+    
+    def test_add_person_added(self):
+        '''
+        tests that all person(s) are added
+        '''
+        self.assertEqual(len(self.dojo.all_people), 0)
+        self.dojo.create_room('brown', 'office')
```

Comment:
I think they are necesary whether they can work with multiple and single entries.